packages/cli/src/login.ts

```diff
@@ -7,60 +7,78 @@ export const loginCommand = async () => {
   const done = Promise.withResolvers<void>();
 
   // start callback server
-  const server = Deno.serve({
-    port: AUTH_PORT_CLI,
-    onListen: () => {
-      const browser = Deno.env.get("BROWSER") ?? "open";
-      const command = new Deno.Command(browser, { args: [DECO_CHAT_LOGIN] });
-      command.spawn();
+  const server = Deno.serve(
+    {
+      port: AUTH_PORT_CLI,
+      onListen: () => {
+        const getBrowserCommand = () => {
+          const userBrowser = Deno.env.get("BROWSER");
+          if (userBrowser) return userBrowser;
+
+          switch (Deno.build.os) {
+            case "linux":
+              return "xdg-open";
+            case "windows":
+              return "start";
+            default:
+              return "open";
+          }
+        };
```

Comment:
nit:

Suggestion only:
```suggestion
        const command = Deno.env.get("BROWSER") ?? { linux: "xdg-open", windows: "start" }[Deno.build.os] ?? "open";
```

Reply:
Done!